lib/one_wire.toit

```diff
@@ -0,0 +1,168 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import rmt
+
+/**
+Support for 1-wire protocol.
+
+The 1-wire protocol is implemented with ESP32's hardware supported RMT module.
+*/
+
+/**
+The 1-wire protocol.
+
+Use $read_bits and $write_bits to read or write bytes to the receiver.
+
+Use $write_then_read to write bytes to the receiver and then immediately start reading.
+
+Use $reset to reset the receiver.
+*/
+class OneWire:
```

Comment:
Should this class be called `Protocol`? This might look better in files that import the lib. For instance
```
import one_wire
[...]
ow := one_wire.Protocol [...]
```